toolkit/src/styles/components/list-item.scss

```diff
@@ -0,0 +1,56 @@
+/**
+* Copyright (c) HashiCorp, Inc.
+*/
+
+@import '@hashicorp/design-system-components';
+
+%cut-list-item {
+  display: flex;
+  flex-direction: row;
+  flex-grow: 0;
+  justify-content: space-between;
+  align-items: center;
+  padding: 12px;
+  border-radius: 6px;
+  gap: 8px;
+}
+
+.cut-list-item {
+  @extend .hds-surface-mid;
+  @extend .hds-elevation-mid;
```

Comment:
I _think_ these two styles are conflicting one with another (one will overwrite the other, I suspect)